custom_components/hacs/__init__.py

```diff
@@ -131,19 +131,18 @@ async def async_startup():
         """HACS startup tasks."""
         hacs.enable_hacs()
 
-        for location in (
-            hass.config.path("custom_components/custom_updater.py"),
-            hass.config.path("custom_components/custom_updater/__init__.py"),
-        ):
-            if os.path.exists(location):
-                hacs.log.critical(
-                    "This cannot be used with custom_updater. "
-                    "To use this you need to remove custom_updater form %s",
-                    location,
-                )
-
-                hacs.disable_hacs(HacsDisabledReason.CONSTRAINS)
-                return False
+        try:
+            import custom_components.custom_updater
```

Comment:
A custom integration could potentially be installed after hacs in the same startup so an import check won't catch all cases. If this is a best effort kind of check it may be ok.

Reply:
As that integration does not have a config flow, a restart will be needed before the user starts using it, and on that startup this will catch it, so that is acceptable here 👍